src/test/scala/org/camunda/feel/impl/interpreter/InterpreterExpressionTest.scala

```diff
@@ -75,4 +75,19 @@ class InterpreterExpressionTest
     eval("null != null") should be(ValBoolean(false))
   }
 
+  it should "compare to not existing variable" in {
```

Comment:
Please add test cases for existing variables which are null

```eval("null = {"x": null}.x) should be(ValBoolean(true))```

Same for property